<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>798</x>
      <y>2</y>
      <width>48</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="minButton">
    <property name="geometry">
     <rect>
      <x>766</x>
      <y>2</y>
      <width>33</width>
      <height>18</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>33</width>
      <height>18</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>33</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="orderButton">
    <property name="geometry">
     <rect>
      <x>734</x>
      <y>2</y>
      <width>33</width>
      <height>18</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>33</width>
      <height>18</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>33</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>135</width>
      <height>18</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>135</width>
      <height>18</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>135</width>
      <height>18</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 0 9pt &quot;黑体&quot;;
font-weight: bold;
color: rgb(150, 150, 150);
</string>
    </property>
    <property name="text">
     <string> 健康管理 </string>
    </property>
   </widget>
   <widget class="QLabel" name="maingraph">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>850</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;黑体&quot;;
color: rgb(255, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="showstatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>96</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="showusername">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>40</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>851</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(44, 44, 44);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="linegraph">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>550</width>
      <height>220</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>110</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="today">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>501</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 18pt &quot;楷体&quot;;
text-decoration: underline;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttondelete">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>52</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>删除当前用户</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/resources/deleteusericon.png);</string>
    </property>
   </widget>
   <widget class="QLabel" name="labeldelete">
    <property name="geometry">
     <rect>
      <x>793</x>
      <y>32</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 75 10pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>删除用户</string>
    </property>
   </widget>
   <widget class="QLabel" name="hint">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0);
font: 75 12pt &quot;宋体&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="between">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>572</y>
      <width>541</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;黑体&quot;;
color: rgb(255, 0, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>379</y>
      <width>221</width>
      <height>215</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="title">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(102, 212, 61);
color: rgb(255, 255, 255);
font: 75 14pt &quot;黑体&quot;;
text-decoration: underline;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="lineWidth">
        <number>4</number>
       </property>
       <property name="text">
        <string>体检表</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="h">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(102, 212, 61);
color: rgb(255, 255, 255);
font: 75 16pt &quot;方正喵呜体&quot;;
</string>
       </property>
       <property name="text">
        <string>身高（单位cm）：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputh"/>
     </item>
     <item>
      <widget class="QLabel" name="w">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(102, 212, 61);
color: rgb(255, 255, 255);
font: 75 16pt &quot;方正喵呜体&quot;;
</string>
       </property>
       <property name="text">
        <string>体重（单位kg）：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputw"/>
     </item>
     <item>
      <widget class="QLabel" name="c">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(102, 212, 61);
color: rgb(255, 255, 255);
font: 75 16pt &quot;方正喵呜体&quot;;
</string>
       </property>
       <property name="text">
        <string>腰围（单位cm）：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputc"/>
     </item>
    </layout>
    <zorder>inputc</zorder>
    <zorder>c</zorder>
    <zorder>inputw</zorder>
    <zorder>w</zorder>
    <zorder>inputh</zorder>
    <zorder>h</zorder>
    <zorder>title</zorder>
   </widget>
   <widget class="QPushButton" name="showresult">
    <property name="geometry">
     <rect>
      <x>283</x>
      <y>380</y>
      <width>461</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/cover);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>359</x>
      <y>380</y>
      <width>381</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="show1">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);
font: 75 11pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>aaa</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="show2"/>
     </item>
     <item>
      <widget class="QLabel" name="show3"/>
     </item>
     <item>
      <widget class="QLabel" name="show4"/>
     </item>
     <item>
      <widget class="QLabel" name="show5"/>
     </item>
     <item>
      <widget class="QLabel" name="show6"/>
     </item>
    </layout>
    <zorder>show6</zorder>
    <zorder>show5</zorder>
    <zorder>show4</zorder>
    <zorder>show3</zorder>
    <zorder>show2</zorder>
    <zorder>show1</zorder>
   </widget>
   <widget class="QTableView" name="tableview">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>151</y>
      <width>261</width>
      <height>197</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>573</x>
      <y>130</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0);
font: 75 10pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>最近十次体检记录(Recent):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>713</x>
      <y>572</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 255);
font: 75 13pt &quot;Aharoni&quot;;</string>
    </property>
    <property name="text">
     <string>@YLAsce</string>
    </property>
   </widget>
   <widget class="QLabel" name="back0">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>380</y>
      <width>461</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/back0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>back0</zorder>
   <zorder>linegraph</zorder>
   <zorder>line</zorder>
   <zorder>closeButton</zorder>
   <zorder>minButton</zorder>
   <zorder>orderButton</zorder>
   <zorder>label</zorder>
   <zorder>showstatus</zorder>
   <zorder>showusername</zorder>
   <zorder>label_2</zorder>
   <zorder>today</zorder>
   <zorder>buttondelete</zorder>
   <zorder>labeldelete</zorder>
   <zorder>hint</zorder>
   <zorder>between</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>showresult</zorder>
   <zorder>tableview</zorder>
   <zorder>label_3</zorder>
   <zorder>maingraph</zorder>
   <zorder>label_4</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
